.clang-format

```diff
@@ -0,0 +1,23 @@
+BasedOnStyle : google
+IndentWidth : 3
```

Comment:
Do you object to "3" or standardizing indentation?  Personally, I prefer 2 instead.